gateleen-delta/pom.xml

```diff
@@ -18,5 +18,13 @@
             <artifactId>gateleen-routing</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <!-- TEST dependencies -->
+        <dependency>
```

Comment:
Is not really used or do I miss something?

Reply:
We get JUnit and Mockito with this dependency. It's `<scope>test</scope>`.
So I don't understand your question?

Reply:
You're right I just saw the AbstractLuaScriptTest class